test/PerfTests/durableJS/FanOutFanIn/function.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
don't love the current naming conventions of `durableJS`, `durablePy`. I would prefer we just have `javascript`, `python` etc, or even `durable-javascript`, `durable-python`.

Reply:
Sounds good, I renamed these directories, as well as some smoke tests which had the `durable-PL` naming scheme.